capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -0,0 +1,95 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+import java.time.LocalDateTime;
+import java.util.*;
+
+@Entity(name = "event")
+public class Event{
+
+  @Id
+  Long datastoreID;
+
+  @Field(name="organizationName")
+  String organizationName;
+
+  @Field(name="eventTitle")
+  String eventTitle;
+
+  @Field(name="eventDateTime")
+  String eventDateTime;
+
+  @Field(name="eventDescription")
+  String eventDescription;
+
+  // @Field(name="university")
+  // String university;
+
+  @Field(name="eventLatitude")
+  double eventLatitude;
+
+  @Field(name="eventLongitude")
+  double eventLongitude;
+
+  // @Field(name="eventFilters")
+  // List<String> eventFilters;
+  
+  // public Event(Long datastoreID, Long organizationID, String organizationName, String eventTitle, LocalDateTime eventDateTime, String university, String eventDescription, Long eventLatitude, Long eventLongitude, List<String> eventFilters) {
+    
+  //   this.datastoreID = datastoreID;
+  //   this.organizationID = organizationID;
+  //   this.organizationName = organizationName;
+  //   this.eventTitle = eventTitle;
+  //   this.eventDateTime = eventDateTime;
+  //   this.eventDescription = eventDescription;
+  //   this.university = university;
+  //   this.eventLatitude = eventLatitude;
+  //   this.eventLongitude = eventLongitude;
+  //   this.eventFilters = eventFilters;
+  // }
```

Comment:
I would start uncommenting these lines of code. Don't worry about conflicts with Lian or I. We have to shift over to the actual event class eventually so better to do it sooner than later.